src/store/level-wrapper.ts

```diff
@@ -112,7 +113,7 @@ export class LevelWrapper<V> {
   async * keys(options?: LevelWrapperOptions): AsyncGenerator<string> {
     options?.signal?.throwIfAborted();
 
-    await abortOr(options?.signal, this.createLevelDatabase());
+    await abortOr(options?.signal!, this.createLevelDatabase());
```

Comment:
`abortOr` actually checks if `signal` is undefined. We should update the method signature to allow undefined, instead of doing `.signal!` here.

Reply:
Done.